src/main/scala/mesosphere/marathon/MarathonSchedulerService.scala

```diff
@@ -258,6 +257,8 @@ class MarathonSchedulerService @Inject() (
 
     schedulerActor ! Suspend(LostLeadershipException("Leadership was defeated"))
 
+    reconciliationTimer.cancel()
```

Comment:
This invalidates the timer, causing any subsequent calls to `schedule` to throw an `IllegalStateException`.  We need to create a new timer at some point.  See: http://docs.oracle.com/javase/7/docs/api/java/util/Timer.html#cancel()